railties/lib/rails/test_unit/minitest_plugin.rb

```diff
@@ -0,0 +1,14 @@
+require "minitest"
+require "rails/test_unit/reporter"
+
+def Minitest.plugin_rails_init(options)
+  self.reporter << Rails::TestUnitReporter.new(options[:io], options)
+  if $rails_test_runner && (method = $rails_test_runner.find_method)
```

Comment:
I see. Since we control the superclass (everything inherits from `AS::TestCase`) we can implement [the runnables method](https://github.com/seattlerb/minitest/blob/master/lib/minitest.rb#L337-L343), so maybe we don't need these globals.